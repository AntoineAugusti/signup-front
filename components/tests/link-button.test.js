import ReactTestRenderer from 'react-test-renderer'
import LinkButton from '../try-me-button'

describe('components | LinkButton', () => {
  describe('render', () => {
    const props = {
      url: 'http://fakeurl.com',
      text: 'fake text'
    }
    const renderer = ReactTestRenderer.create(<LinkButton {...props} />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
