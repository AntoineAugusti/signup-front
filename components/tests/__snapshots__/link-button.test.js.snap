// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | LinkButton render should match snapshot 1`] = `
<a
  className="button"
  href="http://fakeurl.com"
  onClick={[Function]}
>
  Essayez-moi !
</a>
`;
